src/inheritAttributes.js

```diff
@@ -315,12 +315,21 @@ export const inheritAttributes = (mpd, options = {}) => {
     throw new Error(errors.INVALID_NUMBER_OF_PERIOD);
   }
 
+  const locations = findChildren(mpd, 'Location');
+
   const mpdAttributes = parseAttributes(mpd);
   const mpdBaseUrls = buildBaseUrls([ manifestUri ], findChildren(mpd, 'BaseURL'));
 
   mpdAttributes.sourceDuration = mpdAttributes.mediaPresentationDuration || 0;
   mpdAttributes.NOW = NOW;
   mpdAttributes.clientOffset = clientOffset;
 
-  return flatten(periods.map(toAdaptationSets(mpdAttributes, mpdBaseUrls)));
+  if (locations.length) {
+    mpdAttributes.location = getContent(locations[0]);
```

Comment:
I see in the spec that location can be specified `0` to `n` times. I think that means that we could have multiple locations, would it make more sense to follow the spec here and support multiple?

Reply:
I was going to say since we aren't going to support it in VHS right now, what's the point, but making it return an array from here isn't that hard and would mean that we or others wouldn't need an mpd-parser update should we want to start supporting multiple Location elements.

Reply:
Making the change, I'll then update VHS to just get the first one (for now, at least)